account-service/src/main/java/com/bdonor/accountservice/Controller/AccountController.java

```diff
@@ -17,74 +15,57 @@
     public String create( @PathVariable String bloodGroup , @PathVariable  String firstname, @PathVariable  String _surname, @PathVariable  String _email, @PathVariable  String _password, @PathVariable  String _addressline, @PathVariable  String _postcode){
         User CreateUser = Service_functions.create(bloodGroup, firstname,  _surname,  _email,  _password,  _addressline,  _postcode);
         System.out.println("this is working");
         return CreateUser.toString();
     }
 
     @ResponseBody
-    @GetMapping("/getUser/{firstName}") // now OK
-    public String getUser( @PathVariable String firstName ){
+    @GetMapping("/getUser/{firstname}") // Works Partially - Only works for one user within database, if there are more with the same name, error is given
+    public String getUser( @PathVariable String firstname ){
         System.out.println("Working");
-        return Service_functions.getByfirstName(firstName).toString();
+        return Service_functions.getByfirstName(firstname).toString();
     }
 
     @ResponseBody
     @GetMapping("/getAll")
-    public String getAllUsers(){ // THIS DOESNT WORK
+    public String getAllUsers(){ // Works
         System.out.println("This Works");
         return Service_functions.getAll().toString();
     }
 
     @ResponseBody
-    @RequestMapping("/updateUser/{bloodGroup}/{firstname}/{_surname}/{_email}/{_addressline}/[_postcode")
+    @RequestMapping("/updateUser/{bloodGroup}/{firstname}/{_surname}/{_email}/{_password}/{_addressline}/{_password}") // Works Partially - Creates new user instead of updating current
     public String updateUser( @PathVariable String bloodGroup , @PathVariable  String firstname, @PathVariable  String _surname, @PathVariable  String _email, @PathVariable  String _password, @PathVariable  String _addressline, @PathVariable  String _postcode ){
         User Update = Service_functions.Update(bloodGroup, firstname,  _surname,  _email,  _password,  _addressline,  _postcode);
         return Update.toString();
     }
 
     @ResponseBody
-    @GetMapping("/deleteUser/{firstName}")
-    public String delete(@PathVariable("firstName") String firstName ){
-        Service_functions.deleteByfirstName(firstName);
-        return "Deleted" + firstName;
+    @GetMapping("/deleteUser/{firstname}") // Works
+    public String deleteUser(@PathVariable String firstname ){
+        Service_functions.deleteByfirstName(firstname);
+        return "Deleted" + firstname;
     }
 
     @ResponseBody
-    @RequestMapping("/Test")
-    public String Test(@RequestParam String firstName){
-        System.out.println("Working");
-        User aa = Service_functions.getByfirstName(firstName);
-        return "found" + aa.toString();
+    @GetMapping("/checkCredentials/{email}/{_password}") // Error - No property email found for type User! Did you mean '_email'?
+    public boolean checkCredentials( @PathVariable String email, @PathVariable String _password ){
```

Comment:
parameters inconsistent, let's use "_" in class variables only (it's only naming conventions but goes well with Java)